### PR TITLE
PACKMP-12 Add plugin property 'nodeJsMinVersion'

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

**Review by jblievremont — APPROVED** [review]
Looks good on the functional side, I'd love to see a test (e.g an IT similar to the one from PACKMP-11?)

File: src/main/java/org/sonarsource/pluginpackaging/AbstractSonarMojo.java [hunk 1]
```diff
@@ -225,6 +228,11 @@ protected String getJreMinVersion() {
     return jreMinVersion;
   }
 
+  @CheckForNull
+  protected String getNodeJsMinVersion() {
```
Comment (damien-urruty-sonarsource) [inline src/main/java/org/sonarsource/pluginpackaging/AbstractSonarMojo.java:232]: I usually prefer to return an optional in such cases, it's more explicit than the annotation IMO. But it's coherent with the rest of the class

**Review by damien-urruty-sonarsource — APPROVED** [review]
Just a small comment but looks good!

**Comment by henryju** [comment]
> Looks good on the functional side, I'd love to see a test (e.g an IT similar to the one from PACKMP-11?)

Good catch, I already had done it, just forgot to add the files ;)

**Pull request merged**